packages/table/src/table.tsx

```diff
@@ -316,8 +344,16 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
 
         const selectedRegions = (props.selectedRegions == null) ? [] as IRegion[] : props.selectedRegions;
 
+        let focusedCellCoordinates: IFocusedCellCoordinates = null;
```

Comment:
try not to initialize anything to `null` unless required by a 3rd party API, use `undefined` instead